.tool-versions

```diff
@@ -1,3 +1,3 @@
-elixir 1.11.4-otp-23
-erlang 23.3.4.18
+elixir 1.17.3-otp-27
+erlang 27.1.1
 nodejs 16.20.0
```

Comment:
question: should this get updated too, since the container is getting updated?

Reply:
If I do that assets break in a way I didn't want to tackel. I can pull that thread if you like?

Reply:
I elected to update this and make sure the Dockerfile was correct too. See ad51903b44986a41787fca658d047f6c0ee033f2